api/bibliotheca.py

```diff
@@ -700,6 +679,44 @@ def process_batch(self, identifiers):
                 pool.update_availability(0, 0, 0, 0, self.analytics)
                 pool.last_checked = now
 
+    def _process_circulation_data(
```

Comment:
It turns out I didn't really need this helper method, but there's still some things in this method not tested so it seems useful to have it.